_pages/blog.md

```diff
@@ -10,7 +10,7 @@ header:
 author_profile: false
 ---
 
-## Recent pyOpenSci Posts!
+## Recent pyOpenSci Posts
```

Comment:
delete exuberance (breaking change) <3

Reply:
@sneakers-the-rat best review comment ever.